tests/mocha/workspace_svg_test.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+suite('WorkspaceSvg', function() {
+  setup(function() {
+    var toolbox = document.getElementById('toolbox-categories');
+    this.workspace = Blockly.inject('blocklyDiv', {toolbox: toolbox});
+    Blockly.defineBlocksWithJsonArray([{
+      'type': 'simple_test_block',
+      'message0': 'simple test block',
+      'output': null
+    },
+    {
+      'type': 'test_val_in',
+      'message0': 'test in %1',
+      'args0': [
+        {
+          'type': 'input_value',
+          'name': 'NAME'
+        }
+      ]
+    }]);
+  });
+
+  teardown(function() {
+    delete Blockly.Blocks['simple_test_block'];
+    delete Blockly.Blocks['test_val_in'];
+    this.workspace.dispose();
+  });
+
+  test('appendDomToWorkspace alignment', function() {
+    var dom = Blockly.Xml.textToDom(
+        '<xml xmlns="https://developers.google.com/blockly/xml">' +
+        '  <block type="math_random_float" inline="true" x="21" y="23">' +
+        '  </block>' +
+        '</xml>');
+    Blockly.Xml.appendDomToWorkspace(dom, this.workspace);
+    chai.assert.equal(this.workspace.getAllBlocks(false).length, 1,
+        'Block count');
+    Blockly.Xml.appendDomToWorkspace(dom, this.workspace);
+    chai.assert.equal(this.workspace.getAllBlocks(false).length, 2,
+        'Block count');
+    var blocks = this.workspace.getAllBlocks(false);
+    chai.assert.equal(blocks[0].getRelativeToSurfaceXY().x, 21,
+        'Block 1 position x');
+    chai.assert.equal(blocks[0].getRelativeToSurfaceXY().y, 23,
+        'Block 1 position y');
+    chai.assert.equal(blocks[1].getRelativeToSurfaceXY().x, 21,
+        'Block 2 position x');
+    // Y separation value defined in appendDomToWorkspace as 10
+    chai.assert.equal(blocks[1].getRelativeToSurfaceXY().y,
+        23 + blocks[0].getHeightWidth().height + 10,
```

Comment:
I added comment to explain where the 10 is coming from, but not whether the 10 in `appendDomToWorkspace` should be moved out into a constant.

Reply:
It's fine to leave it as-is.  It's not a parameter I want to make tweakable.